scripts/deploy.sh

```diff
@@ -56,12 +56,14 @@ if [[ "${DEPLOY_AIO}" == "yes" ]]; then
   export DEPLOY_HAPROXY="yes"
   if [[ ! -d /etc/openstack_deploy/ ]]; then
     ./scripts/bootstrap-aio.sh
+    # move OSA conf files to correct locations
+    mv /etc/openstack_deploy/user_variables.yml /etc/openstack_deploy/user_osa_variables_defaults.yml
```

Comment:
I think its confusing to rename files from OSA. It will be difficult to trace where user_osa_variables_defaults came from because its not in the OSA or RPC repos.